ci/azure_pipelines/templates/artifacts.yml

```diff
@@ -0,0 +1,120 @@
+# This template includes the steps for collecting
+# logs and files generated during the build process
+
+steps:
+- task: ComponentGovernanceComponentDetection@0
+  inputs:
+    scanType: 'Register'
+    verbosity: 'Verbose'
+    # dockerImagesToScan: 'docker-image'
+    alertWarningLevel: 'Critical'
+    failOnAlert: true
+    # ignoreDirectories: '**/_bazel_runner'
```

Comment:
Intended to be commented out? (Or perhaps meant to ignore all bazel-* directories?)

Reply:
Good catch. I forgot this.

The first comment line about docker, it's just a place holder to know where to configure docker images that have to be reviewed by the governance task. I haven't identified one yet. But given the fact that this is not documented and it was tricky to find the options, I will put this commented out but with empty value.

Regarding the other line, I'm not 100% sure what is the right thing to do here: if we should ignore everything that Bazel is pulling in order to build faster or let Component Governance be aware of everything. I chose the last. But I will do the same that above: comment out and put empty value.

Thx